client/src/components/ProfilePictures.js

```diff
@@ -0,0 +1,15 @@
+import styled from "@emotion/styled";
+
+const ProfilePicture = styled.img`
+  border-radius: 100px;
+  height: ${(props) => profilePictureSizes[props.size].height};
+  width: ${(props) => profilePictureSizes[props.size].width};
+`;
+
+const profilePictureSizes = {
+  small: { height: "16px", width: "16px" },
+  medium: { height: "40px", width: "40px" },
+  big: { height: "60px", width: "60px" },
+};
```

Comment:
If your profile pictures are always "squares" with the same `height` and `width`, you only need one property.

Reply:
You're right, thanks! I thought I need both properties.